src/alpaca_eval/leaderboards/data_AlpacaEval/alpaca_eval_gpt4_leaderboard.csv

```diff
@@ -1,63 +1,64 @@
 ,win_rate,standard_error,n_wins,n_wins_base,n_draws,n_total,mode,avg_length
```

Comment:
it's surprising that everything changed. I think I forgot to update the leaderboard to use integer avg_length.

Reply:
maybe it is because I pulled the latest code but inserted the single row's evaluation result of our model from last week's  leaderboard.csv to the latest version's leaderboard.csv. 
thanks for your merge!! :hugs: